sdk/clientcore/core/src/main/java/io/clientcore/core/implementation/http/rest/SwaggerMethodParser.java

```diff
@@ -327,10 +362,19 @@ public String setPath(Object[] methodArguments, ObjectSerializer serializer) {
     @SuppressWarnings("unchecked")
     public void setEncodedQueryParameters(Object[] swaggerMethodArguments, UrlBuilder urlBuilder,
                                           ObjectSerializer serializer) {
+        // First we add the constant query parameters.
+        for (Map.Entry<String, List<String>> queryParam : constantQueryParams.entrySet()) {
+            for (String value : queryParam.getValue()) {
+                addSerializedQueryParameter(serializer, value, false, urlBuilder, queryParam.getKey());
```

Comment:
This should use `urlBuilder.addQueryParameter` directly as

1. We shouldn't need to serialize the header value as the annotation only allows string values, so there is no object value to serialize.
2. We should already have the query parameter value encoded when we parse them as constants.